core/src/main/java/io/questdb/cairo/sql/VirtualRecord.java

```diff
@@ -129,13 +129,18 @@ public long getLong(int col) {
     }
 
     @Override
-    public long getLong128Hi(int col) {
-        return getFunction(col).getLong128Hi(base);
+    public long getLong128Hi(int col, long location) {
+        return getFunction(col).getLong128Hi(base, location);
```

Comment:
this escalates :) `location` is the `address` nothing other than `getLong()` is needed here